config/cucumber.yml

```diff
@@ -1,4 +1,5 @@
 default: -t "not @benchmarking" -t "not @pending" -t "not @local-network" -t "not @notintegration"
+test: -t "not @pending" -t "not @nottest" -t "not @notaws"
```

Comment:
Should this mention the `@replatforming` tag you added somewhere? Or do you want that to be explicit?

Reply:
Yeah I'd prefer it to be explicit. Partially for consistency, but also since we can use the `replatforming` tag against other environments, which might come in handy when testing against integration.